tidbcloud: update changefeed replication (#8837)
pingcap/docs: tidb-cloud/changefeed-replication.md

- Identify all dependent services, check the guarantees and capabilities of these services, and ensure that your application is operational during a failover of these services.

## Limitations

Currently, the **TiDB Cloud Replication** feature is in **Public Preview** with the following limitations:

* One primary cluster can only have one replication.
* You cannot use a secondary cluster as a source of **TiDB Cloud Replication** to another cluster.
* **TiDB Cloud Replication** contradicts [**Sink to Apache Kafka**](/tidb-cloud/changefeed-sink-to-apache-kafka.md) and [**Sink to MySQL**](/tidb-cloud/changefeed-sink-to-mysql.md). When **TiDB Cloud Replication** is enabled, neither the primary nor the secondary cluster can use the **Sink to Apache Kafka** or **Sink to MySQL** changefeed and vice versa.
* Because TiDB Cloud uses TiCDC to establish replication, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#restrictions).

> * The network latency between the primary and secondary clusters affects the performance or stability of **TiDB Cloud Replication**. with the following limitations:

Reviewer: ```suggestion
* Depending on the network latency between the primary and secondary clusters, the performance and stability of **TiDB Cloud Replication** are affected as follows:
```

>   * The network latency less than 120 millisecond is recommended.

Reviewer: ```suggestion
  * To achieve the best performance and stability of **TiDB Cloud Replication**, the network latency within 120 milliseconds is recommended.
```
